k4GaudiPandora/include/DDSimpleMuonDigi.h

```diff
@@ -72,18 +64,16 @@ struct DDSimpleMuonDigi final
   Gaudi::Property<float>       m_timeThresholdMuon{this, "timethresholdMuon", {0.025}, "time threshold for muons"};
   Gaudi::Property<float>       m_calibrCoeffMuon{
       this, "calibrationCoeffmuon", {120000.0}, "Callibration coefficient of muons"};
-  Gaudi::Property<float> m_maxHitEnergyMuon{this, "maxMuonHitEnergy", {2.0}, "Threshold for maximum muon hit energy"};
+  Gaudi::Property<float> m_maxHitEnergyMuon{this, "MaxMuonHitEnergy", {2.0}, "Threshold for maximum muon hit energy"};
```

Comment:
Some other properties also start with a lowercase character. What is the motivation to change this one?